components/Containers/NavBar/NavItem/index.module.css

```diff
@@ -24,6 +24,13 @@
         dark:text-white;
     }
 
+    &:not(.active):hover {
+      .label {
+        @apply text-green-400
```

Comment:
The contrast ratio does not seem sufficient for the light theme;

<img width="663" alt="image" src="https://github.com/nodejs/nodejs.org/assets/14062599/a5355195-f38a-40cd-905b-d357790a0f6c">

Reply:
I'll fix that, what's the tool/extension name? so i can test out the Contrast of the color i put

Reply:
Thank you, i'll be fixing this shortly.

Reply:
Changed it to text-green-700 which has 5.01 contrast.